### PR TITLE
Render views

### PR DESCRIPTION
**Pull request by NatanimA**
### In this PR:
- Implemented the design from the sneak peek wireframes.

    
sneak2

sneak1

sneak3

sneak 4

---
### To Run the Application please follow the following

- [ ] Configure your Database

blog_app_test

- [ ] Make sure you have data to be rendered in your database
- [ ] Run ```rails s```

### CODE REVIEW

File: app/views/users/_user.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="user-list-container">
+    <div class="user-image-container">
+         <image class="user-photo" src="<%= user.photo %>"></image>
+    </div>
+    <div class="user-detail-container">
+        <h3 class="user-name"><%= user.name %></h3>
+        <span class="num-posts">Number of posts: <%= user.posts_counter %></span>
+    </div>
+</div>
```
Comment (julie-ify) [inline app/views/users/_user.html.erb:10]: - [x] Clicking on a user should take me to their corresponding show page. Kindly make this user name or div class a clickable link that fulfills the above specification.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -1,10 +1,10 @@
 class UsersController < ApplicationController
   def index
-    render 'index'
+    @user = User.all
   end
 
   def show
-    render 'show'
+    @post = User.recent_posts(params[:id])
```
Comment (julie-ify) [inline app/controllers/users_controller.rb]: - [ ] **[OPTIONAL]** We can improve the `recent_posts` scope in the User model and make it just a method. Then we can call it here. So we don't need to be using `posts[0]` in the views

  ```
  def recent_posts
    posts.order(created_at: :desc).limit(5)
  end
  ```

  This way we can just call the users' posts in the view.

  ```
  @user = User.find(params[:id)
  ```

  Then in the show view we can present the recent posts:

  ```
  @user.recent_posts
  ```

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -1,107 +1,15 @@
...
-         Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
-         mollit anim id est laborum.
-    </p>
-    <div class="interaction-container">
-        <span class="comments-counter">Comments X</span>
-        <span class="likes-counter">Likes Y</span>
-    </div>
+<%= render @post %>
+<div class="btn-container-post">
+<button class="btn-posts">See all Posts</button>
```
Comment (julie-ify) [inline app/views/users/show.html.erb:14]: - [x] Kindly make this button into a link that navigates to the post index page for the user.

File: app/views/posts/index.html.erb [hunk 4]
```diff
@@ -1,104 +1,17 @@
...
-    <div class="comments-list-container">
-        <span class="commenter-user-name">Username:</span>
-        <p class="comments-content">Comment <%= Random.rand(1..20) %></p>
-    </div>
-    <div class="comments-list-container">
-        <span class="commenter-user-name">Username:</span>
-        <p class="comments-content">Comment <%= Random.rand(1..20) %></p>
-    </div>
-
+    <%= render post.comments %>
```
Comment (julie-ify) [inline app/views/posts/index.html.erb]: - [x] You should use the recent comments method in the post model for this loop instead of rendering all comments.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @NatanimA,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @NatanimA,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**